docs/integrations/builtin/request-builtin-node.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Request a built-in node
+pageType: howto
+description: How to request and submit built-in nodes for n8n. 
+---
+
+# Request a built-in node
+
+## Make a feature request
+
+The best way to request a built-in node is to post it in the n8n Community Forum. You can upvote an existing post or create a new post in the [Feature Request > Nodes category](https://community.n8n.io/c/feature-requests/nodes/){:target=_blank .external-link}.
+
+## Submit a built-in node
+
+If you've created a node you'd like to add as an official node in n8n, open a pull request on [GitHub](https://github.com/n8n-io/n8n){:target=_blank .external-link}. An n8n team member will review it. Note that new nodes take time to review and this may take some time. 
```

Comment:
Can we remove this? Any new node PRs are closed with a message to direct them to comunity nodes.